swagger.yaml

```diff
@@ -436,6 +436,31 @@ paths:
           description: "Internal Server Error"
           schema:
             $ref: "#/definitions/Error"
+  /feed:
+    get:
+      tags:
+        - feed
+      description: "Gets a CSV feed of events to be added to the Localist calendar"
+      parameters:
+        - name: changedInPastHours
+          in: query
+          description: "Feed will only contain events created or changed within this many past hours"
+          type: integer
+      produces:
+        - application/csv
```

Comment:
Can you change this to `text/csv` to reflect that the feedresource is returning?